systemtest/src/test/java/io/strimzi/systemtest/StrimziUpgradeST.java

```diff
@@ -90,15 +95,29 @@ void upgradeStrimziVersion(JsonObject parameters) throws Exception {
             kafkaUserYaml = new File(dir, parameters.getString("fromExamples") + "/examples/user/kafka-user.yaml");
             cmdKubeClient().create(kafkaUserYaml);
 
+            // Wait until user will be created
+            // We cannot use utils wait for that, because in older version there were no status field for CRs
+            Thread.sleep(10000);
```

Comment:
We should use status check instead of timeout.

Reply:
It's not possible since CR status is not available in all versions which we have in upgrade test

Reply:
Ok, good point I guess.